examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1290,38 +1290,40 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
             text_encoder_lora_layers=text_encoder_lora_layers,
             text_encoder_2_lora_layers=text_encoder_2_lora_layers,
         )
+        
 
-        # Final inference
-        # Load previous pipeline
-        vae = AutoencoderKL.from_pretrained(
-            vae_path,
-            subfolder="vae" if args.pretrained_vae_model_name_or_path is None else None,
-            revision=args.revision,
-            torch_dtype=weight_dtype,
-        )
-        pipeline = StableDiffusionXLPipeline.from_pretrained(
-            args.pretrained_model_name_or_path, vae=vae, revision=args.revision, torch_dtype=weight_dtype
-        )
 
-        # We train on the simplified learning objective. If we were previously predicting a variance, we need the scheduler to ignore it
-        scheduler_args = {}
+        images = []
```

Comment:
Shouldn't this also go inside the if condition? We don't need this list otherwise, no?

Reply:
The push_to_hub function below needs it. I updated the PR to initialize it to None instead, since save_model_card accepts that too.